invenio_drafts_resources/services/records/components/files.py

```diff
@@ -130,6 +131,15 @@ def publish(self, identity, draft=None, record=None):
                     ),
                     field_name="files.enabled",
                 )
+        if draft.files.enabled:
+            for file_record in draft.files.values():
+                if not TransferType(file_record.file.storage_class).is_completed:
```

Comment:
I think something went wrong int he rebase here. There are two messages (should only have the second one), also is missing the `field_name="files"`